File: mycollab-web/src/main/java/com/esofthead/mycollab/module/project/view/page/PageListView.java
package com.esofthead.mycollab.module.project.view.page;

import java.util.List;

import com.esofthead.mycollab.module.wiki.domain.WikiResource;
import com.esofthead.mycollab.vaadin.mvp.PageView;

/**
 * 
 * @author MyCollab Ltd.
 * @since 4.4.0
 *
 */
public interface PageListView extends PageView {
	/**
	 * 
	 * @param resources
	 */
	void displayDefaultPages(List<WikiResource> resources);
}

File: mycollab-web/src/main/java/com/mycollab/module/project/view/page/PageListView.kt
package com.mycollab.module.project.view.page

import com.mycollab.module.page.domain.PageResource
import com.mycollab.vaadin.mvp.PageView

/**
 * @author MyCollab Ltd.
 * @since 4.4.0
 */
interface PageListView : PageView {
    /**
     * @param resources
     */
    fun displayDefaultPages(resources: List<PageResource>)

    fun showNoItemView()
}
